lib/exe/css.php

```diff
@@ -34,98 +35,125 @@ function css_out(){
 
     if ($INPUT->str('s') == 'feed') {
         $mediatypes = array('feed');
-        $type = 'feed';
     } else {
-        $mediatypes = array('screen', 'all', 'print');
-        $type = '';
+        $mediatypes = array('screen', 'all', 'print', 'handheld');
```

Comment:
I added handheld as it was proposed at https://bugs.dokuwiki.org/1987.html - The discussion over at #1002 was about automatically adding new media types or a static list. A static list gives more control and adding new media types is ok I guess.